arrus/core/devices/us4r/us4oem/Us4OEMImpl.h

```diff
@@ -223,6 +225,11 @@ class Us4OEMImpl : public Us4OEMImplBase {
     std::vector<IRQEvent> irqEvents = std::vector<IRQEvent>(Us4OEMDescriptor::MAX_IRQ_NR+1);
     /** Max TX pulse length [s]; nullopt means to use up to 32 periods (OEM legacy constraint) */
     std::optional<float> maxPulseLength = std::nullopt;
+    /** DDC extra gain to apply, Currently, simply translates to the boolean value 'gain is off/on'.*/
+    const ValueMap<float, bool> DDC_GAIN_MAP{{
```

Comment:
does not build on windows - conversion from '_Ty' to '_Ty1', possible loss of data